pxfish/pyfish.py

```diff
@@ -111,22 +113,22 @@ def main():
     # have category, look for library or operation type
     category_path = create_named_path(path, args.category)
 
+    user_id = aq.User.where({"login": aq.login}) 
+    
     if args.action == 'create' and args.operation_type:
         create_new_operation_type(aq, path, args.category, args.operation_type)
         get_operation_type(aq, path, args.category, args.operation_type)
         return
 
     if args.library:
-        select_library(aq, category_path, args.library)
+        select_library(aq, user_id, category_path, args.library)
```

Comment:
the user information is part of the session object that you pass in, so it would be better to not pass it in here.
if the issue is efficiency, maybe something in pydent needs to change